pom.xml

```diff
@@ -248,10 +244,88 @@
   <build>
     <pluginManagement>
       <plugins>
+        <plugin>
```

Comment:
Can you give a little context on why you've decided to add `maven-enforcer-plugin` and `versions-maven-plugin`?

Do these replace `maven-jar-plugin` and `maven-compiler-plugin`?

Reply:
Added `maven-enforcer-plugin` to enforce maven version. It allows to ensure that plugin version used by the build are compatible with the maven as certain plugins only support maven starting with a specific version. 

'versions-maven-plugin` is not added, it already comes from the maven pom model. The change here is to ignore SNAPSHOT, alpha and beta versions when running versions reports.

No, `maven-jar-plugin` and `maven-compiler-plugin` are not replaced.

Reply:
Can we add a short note about these 2 changes to the PR description?

Reply:
Done.